backend/src/models/Settings.js

```diff
@@ -1,10 +1,23 @@
 import mongoose from 'mongoose';
+import Joi from 'joi';
 import { LIGHT_THEME, DARK_THEME } from '../constants/themes';
 
 const settingsSchema = new mongoose.Schema({
   appTheme: { type: String, enum: [LIGHT_THEME, DARK_THEME], default: LIGHT_THEME },
   soonExpiringFilterLength: { type: Number, enum: [3, 5, 7], default: 3 },
 });
 
+export const settingsValidator = (req, res, next) => {
+  const schema = Joi.object({
+    appTheme: Joi.string().valid(LIGHT_THEME, DARK_THEME),
+    soonExpiringFilterLength: Joi.number().valid(3, 5, 7),
```

Comment:
Let's extract [3,5,7]  to the single variable in Settings.js, so we don't need to remember about adding another value in 2 places. 

Same can be done with appTheme enum